gitlens/gitlens-release-notes-current.md

```diff
@@ -19,6 +19,98 @@ Check out our [Changelog](https://github.com/gitkraken/vscode-gitlens/blob/main/
 
 ***
 
+<a id="v13-5"></a>
+## Version 13.5
+
+### Thursday, Apr 6, 2023
+
+With 13.5, we've added the ability to switch to an alternate panel layout for the Commit Graph, which moves the Commit Graph into the bottom panel and adds a new Commit Graph Details view alongside on the right, for more convenient and persistent access. You can now also switch to a compact layout for the Graph column and shrink the Branch / Tag column to a single icon to reduce the space and allow you to focus on what you deem most important. You can now take action on your pull requests in the Focus View, with the new ability to create or switch branches or worktrees. We've also heard your feedback that the Commit Details view pinning was confusing and hard to work with, so 13.5 includes many improvements to make that experience better. We've also continued our quest to reduce the size and improve performance of GitLens and have shaved another ~7% off the GitLens bundle.
+
+<img src="/wp-content/uploads/gl-13.5-hero.png" class="img-responsive center img-bordered">
+
+### ✨ Commit Graph Layouts
+
+<img src="/wp-content/uploads/gl-commit-graph-panel-layout-view.png" class="img-responsive center img-bordered">
+
+The new panel layout brings even more flexibility and customization options to GitLens. While using the panel layout, GitLens moves the Commit Graph into the bottom panel and adds a new Commit Graph Details view alongside on the right. If the layout in the bottom panel doesn't suit your needs you can move the Commit Graph around to other locations (side bar, secondary side bar, etc) to suit your preferences. Whether you prefer to have the Commit Graph in the editor or in a separate panel, GitLens makes it easy to switch and find the layout that works best for you.
+
+<img src="/wp-content/uploads/gl-commit-graph-panel-layout-change.gif" class="img-responsive center img-bordered">
+
+To switch to the panel layout, simply click on the Commit Graph settings cog located at the top right of the Commit Graph. From there, select the "Switch Commit Graph to Panel Layout" option, and if you want to switch back follow the same steps and choose "Switch Commit Graph to Editor Layout".
+
+#### Compact Graph Column Layout
```

Comment:
Should this, Pull Requests on Local Branches, and Publish Local Branches from Commit Graph include a ✨?

Reply:
✨ means a GitLens+ feature, are all these GitLens+ features?

Reply:
They are, but I'm not sure we need to add it to each sub-heading, since they are all under the main "Commit Graph Layouts" section that has it

Reply:
Yea should be just on:
## ✨ Commit Graph Layouts
## ✨ Focus View Pull Request Actions